android/src/fdroid/play/listings/ar/release-notes.txt

```diff
@@ -1,13 +1,13 @@
-• New OpenStreetMap data as of May 3
-• Fix search for interpolated odd/even house numbers
-• Fix zoom buttons setting
-• Faster rendering of subway layer
-• Fix wrong postcodes in Paris
-• Fix missing Wikipedia links
-• Fix car routing via roundabouts
-• Enable TTS for Portuguese/Brazil, Español/México
-• Fix location detection over a network on some devices
-• Change gates and Lyon subway icon
-• Fix US road shields
-• Add camera, interior_decoration, antiques, art, cheese shops
+• GPX import support
+• New OpenStreetMap data as of June 2
+• UTM and MRGS coordinates support (tap on coordinates to change)
+• Improved car and bicycle routing
+• Fixed OSM edits upload only on the device's charging
```

Comment:
This would be hard to understand if you didn't know.  I think the long story is that there used to be a bug where OSM edits would only get uploaded if the phone is "on ac power", instead of "on battery".   I realize we have to use few words, so I would say one of

  - Fix bug that prevented upload of OSM edits when on battery power
  - OSM edits are now uploaded promptly, rather than only uploaded when charging

Reply:
Thanks! Fixed.